akka-actor-tests/src/test/scala/akka/actor/SupervisorHierarchySpec.scala

```diff
@@ -160,11 +160,11 @@ object SupervisorHierarchySpec {
           val sizes = s / kids
           var rest = s % kids
           val propsTemplate = Props.empty.withDispatcher("hierarchy")
-          (1 to kids).map { (id) ⇒
+          (1 to kids).iterator.map { (id) ⇒
             val kidSize = if (rest > 0) { rest -= 1; sizes + 1 } else sizes
             val props = Props(new Hierarchy(kidSize, breadth, listener, myLevel + 1, random)).withDeploy(propsTemplate.deploy)
             (context.watch(context.actorOf(props, id.toString)).path, kidSize)
-          }(collection.breakOut)
+          }.toMap
```

Comment:
should this be to(Map)?

Reply:
I think both will work, and `.toMap` could be overridden to provide a more performant implementation compared to the more generic `.to(Map)`, so it might be OK? Can you think of a reason to prefer `.to(Map)`?

Reply:
The reason is just the migration instructions in https://github.com/scala/collection-strawman/wiki/FAQ#what-are-the-breaking-changes:

Description | Old Code | New Code 
-- | -- | -- 
collection.breakOut no longer exists | `val xs: List[Int] = ys.map(f)(collection.breakOut)` | `val xs = ys.iterator.map(f).to(List)`

but in other examples they also use toMap, so maybe it just doesn't matter...